packages/core/src/awsService/appBuilder/serverlessLand/serverlessLandWizard.ts

```diff
@@ -0,0 +1,184 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import * as AWS from '@aws-sdk/types'
+import * as vscode from 'vscode'
+import { Wizard } from '../../../shared/wizards/wizard'
+import * as path from 'path'
+import { createInputBox } from '../../../shared/ui/inputPrompter'
+import { createCommonButtons } from '../../../shared/ui/buttons'
+import { createQuickPick } from '../../../shared/ui/pickerPrompter'
+import { createFolderPrompt } from '../../../shared/ui/common/location'
+import { Region } from '../../../shared/regions/endpoints'
+import { createExitPrompter } from '../../../shared/ui/common/exitPrompter'
+import { MetadataManager } from './serverlesslandMetadataManager'
+import { ToolkitError } from '../../../shared/errors'
+
+export interface CreateServerlessLandWizardForm {
+    name: string
+    location: vscode.Uri
+    pattern: string
+    runtime: string
+    iac: string
+    region?: string
+    schemaName?: string
```

Comment:
What are these two fields? Are they relevant here?

Reply:
No, I kept them as option to see if I will require them later at some place.